pkg/file/file.go

```diff
@@ -0,0 +1,65 @@
+package file
```

Comment:
should the package be `file` or `utils`?

Reply:
util ...correcting this